mne/io/eyetrack/tests/test_eyetrack.py

```diff
@@ -0,0 +1,24 @@
+import os.path as op
+
+from mne.io import read_raw_eyelink
+
+from mne.datasets.testing import data_path, requires_testing_data
+
+testing_path = data_path(download=False)
+fname = op.join(testing_path, 'eyetrack', 'test_eyelink.asc')
+
+raw = read_raw_eyelink(fname, interpolate_missing=True, annotate_missing=True)
```

Comment:
was `interpolate_missing=True` and `annotate_missing=True` working for you? I think I may have broke someting becuase I get a circular import error when trying to import your functions from `mne.preprocessing`!